core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -70,13 +70,16 @@
 import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.TabletColumnFamily;
 import org.apache.accumulo.core.tabletserver.log.LogEntry;
 import org.apache.accumulo.core.util.HostAndPort;
+import org.apache.accumulo.core.util.Pair;
 import org.apache.accumulo.core.util.ServerServices;
 import org.apache.hadoop.io.Text;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
+import com.google.common.base.Supplier;
```

Comment:
This is almost certainly going to trigger modernizer with a suggestion to use `java.util.function.Supplier`. It is possible to use Guava `Suppliers` with the Java functional API, though. I think we have cases where we do that, so it should be easy to change this.

Reply:
Yeah that should be the java version.  I think my IDE auto-magically did that when I used Suppliers.  I changed it in b2a8a32